hugo/layouts/partials/share.html

```diff
@@ -0,0 +1,35 @@
+<details class="share" data-dropdown>
+    <summary class="share__container">
```

Comment:
the naming is a bit weird since this container only contains the button
also in the share.scss I see styles set to list-reset but this is not a list? so maybe both styling and clsasname can be removed and button classnames can be lifted to summary since the a itself doesnt do anything (see comment below)